pom.xml

```diff
@@ -397,14 +397,15 @@
                 <module>security/https</module>
                 <module>security/jwt</module>
                 <module>security/keycloak</module>
-                <module>security/keycloak-authz</module>
+                <module>security/keycloak-authz-classic</module>
                 <module>security/keycloak-jwt</module>
                 <module>security/keycloak-webapp</module>
                 <module>security/keycloak-oauth2</module>
                 <module>security/keycloak-multitenant</module>
                 <module>security/keycloak-oidc-client-basic</module>
                 <module>security/keycloak-oidc-client-extended</module>
                 <module>security/vertx-jwt</module>
+                <module>security/keycloak-authz-reactive-resteasy</module>
```

Comment:
I would drop the word `resteasy`. Simply: reactive. And also group both modules together, pls.